docker-compose.yml

```diff
@@ -1,69 +1,128 @@
-# NOTE: This docker-compose.yml is meant to be just an example of how
-# you could accomplish this on your own. It is not intended to work in
-# all use-cases and must be adapted to fit your needs. This is merely
-# a guideline.
-
-# See docs.getsentry.com/on-premise/server/ for full
-# instructions
-
 version: '3.4'
-
-x-defaults: &defaults
+x-defaults:
   restart: unless-stopped
-  build:
+  build: &ref_0
     context: .
     args:
       SENTRY_IMAGE: ${SENTRY_IMAGE}
-  depends_on:
+  depends_on: &ref_1
     - redis
     - postgres
     - memcached
     - smtp
+    - snuba
+    - symbolicator
+    - kafka
   env_file: .env
-  environment:
+  environment: &ref_2
     SENTRY_MEMCACHED_HOST: memcached
     SENTRY_REDIS_HOST: redis
     SENTRY_POSTGRES_HOST: postgres
     SENTRY_EMAIL_HOST: smtp
-  volumes:
-    - sentry-data:/var/lib/sentry/files
-
-
+    SNUBA: 'http://snuba:1218'
+    SENTRY_KAFKA_HOST: 'kafka:9092'
+  volumes: &ref_3
+    - 'sentry-data:/var/lib/sentry/files'
 services:
   smtp:
     restart: unless-stopped
     image: tianon/exim4
-
   memcached:
     restart: unless-stopped
-    image: memcached:1.5-alpine
-
+    image: 'memcached:1.5-alpine'
   redis:
     restart: unless-stopped
-    image: redis:3.2-alpine
-
+    image: 'redis:5.0-alpine'
+    command: redis-server --appendonly yes
+    volumes:
+      - 'sentry-redis:/data'
   postgres:
     restart: unless-stopped
-    image: postgres:9.5
+    image: 'postgres:9.6-alpine'
+    volumes:
+      - 'sentry-postgres:/var/lib/postgresql/data'
+  zookeeper:
+    image: 'confluentinc/cp-zookeeper:5.1.2'
+    environment:
+      ZOOKEEPER_CLIENT_PORT: '2181'
+    volumes:
+      - 'sentry-zookeeper:/var/lib/zookeeper'
+  kafka:
+    depends_on:
+      - zookeeper
+    image: 'confluentinc/cp-kafka:5.1.2'
+    environment:
+      KAFKA_ZOOKEEPER_CONNECT: 'zookeeper:2181'
+      KAFKA_LISTENERS: 'INTERNAL://kafka:9093,EXTERNAL://kafka:9092'
+      KAFKA_ADVERTISED_LISTENERS: 'INTERNAL://kafka:9093,EXTERNAL://kafka:9092'
+      KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: 'INTERNAL:PLAINTEXT,EXTERNAL:PLAINTEXT'
+      KAFKA_INTER_BROKER_LISTENER_NAME: INTERNAL
+      KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: '1'
+    volumes:
+      - 'sentry-kafka:/var/lib/kafka'
+  clickhouse:
+    image: 'yandex/clickhouse-server:19.3'
+    ulimits:
+      nofile:
+        soft: 262144
+        hard: 262144
     volumes:
-      - sentry-postgres:/var/lib/postgresql/data
-
+      - 'sentry-clickhouse:/var/lib/clickhouse'
+  snuba:
+    depends_on:
+      - redis
+      - clickhouse
+      - kafka
+    restart: unless-stopped
+    image: 'getsentry/snuba:latest'
+    environment:
```

Comment:
All of this is going to be wrong for "production". This mirrors what we have as local dev.

Reply:
Okay, that's what I feared. Any pointers?

Reply:
We should sync up on what I h ave in single-tenant.